NAMESPACE

```diff
@@ -220,6 +220,8 @@ export(kurtosis)
 export(normalize)
 export(object_has_names)
 export(object_has_rownames)
+export(old_data_to_long)
+export(old_data_to_wide)
```

Comment:
I am assuming you are retaining these only to show the benchmarks, and they will be removed before merging the PR?

Since the API hasn't changed, it will be good not to export these old, slower versions.

Reply:
Yes, just for benchmarks